### PR TITLE
Feat/example story

### PR DESCRIPTION
**Pull request by EnoF**

### CODE REVIEW

File: examples/embedded-nextjs/src/steps/Connect.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+              >
+                <SelectItem key="mainnet01">Mainnet</SelectItem>
+                <SelectItem key="testnet04">Testnet</SelectItem>
+                <SelectItem key="development">Devnet</SelectItem>
+              </Select>
+            </Stack>
+          </CardContentBlock>
+        ) : null}
+      </CardContainer>
+      <CardFooter>
```
Comment (eileenmguo) [inline examples/embedded-nextjs/src/steps/Connect.tsx:114]: This should probably be in the CardContainer

File: examples/embedded-nextjs/src/steps/Stepper/Stepper.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      className={styles.stepper}
+      marginInlineStart="sm"
+      flexDirection="column"
+    >
+      {children}
+    </Stack>
+  );
+};
+type StepProps = {
+  status?: 'positive' | 'warning' | 'error' | 'todo';
```
Comment (eileenmguo) [inline examples/embedded-nextjs/src/steps/Stepper/Stepper.tsx:21]: nit: I dont' think status is used?

File: examples/embedded-nextjs/src/steps/Connect.tsx (hunk 1 above)
Comment (EnoF) [reply examples/embedded-nextjs/src/steps/Connect.tsx:114]: No it actually was meant like this, the primary buttons are supposed to be outside of the card (checked with Isa)

File: examples/embedded-nextjs/src/steps/Stepper/Stepper.tsx (hunk 2 above)
Comment (EnoF) [reply examples/embedded-nextjs/src/steps/Stepper/Stepper.tsx:21]: that's correct, will only implement them in next pr

**Pull request merged**